scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/core/properties/PASchedulerProperties.java

```diff
@@ -564,6 +564,10 @@ negative number indicates that session is infinite (value specified in second) *
 
     public static final String PA_SCHEDULER_PROPERTIES_RELATIVE_FILEPATH = "config/scheduler/settings.ini";
 
+    public static final String LABEL_REGEX = "^[a-zA-Z0-9_/-]*$";
+
+    public static final int LABEL_MAX_LENGTH = 20;
```

Comment:
you need to define a enum value (check the other properties defined in this enum class).

Also, you need to add it in config/settings.ini